app/src/main/kotlin/co/netguru/baby/monitor/client/common/base/BaseFragment.kt

```diff
@@ -2,15 +2,27 @@ package co.netguru.baby.monitor.client.common.base
 
 import android.os.Bundle
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
-import androidx.fragment.app.Fragment
+import co.netguru.baby.monitor.client.feature.analytics.AnalyticsManager
+import dagger.android.support.DaggerFragment
+import javax.inject.Inject
 
-abstract class BaseFragment : Fragment() {
+abstract class BaseFragment : DaggerFragment(), AnalyticScreen {
     abstract val layoutResource: Int
+    override val screenName: String? = null
+
+    @Inject
+    lateinit var analyticsManager: AnalyticsManager
 
     override fun onCreateView(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a540cb343de5354d91c6cd7978445c3390e2e9c1/app/src/main/kotlin/co/netguru/baby/monitor/client/common/base/BaseFragment.kt#L18"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>